tests/functional/commands/test_status_command.py

```diff
@@ -93,6 +93,35 @@ def test_experiment_wout_success_wout_ac(clean_db, single_without_success, capsy
     assert captured == expected
 
 
+def test_experiment_number_same_list_status(clean_db, only_invalidalgo_experiments_db,
+                                            single_without_success, capsys):
+    """Test status and list command output the consistent number of experiments"""
```

Comment:
Could we make separate tests for invalidalgo and coherence with list or both are related? I believe they are not.

Reply:
sure, we can, I do it in one as we did not really fix the invalidalgo problem but just make their output consistent, so the invalidalgo and coherence get same output. But once we fix the invalidalgo experiment construct issue, they should be 2 cases.